diff --git a/examples/mnist_mlp.py b/examples/mnist_mlp.py
index cba180ff..575ab0f9 100644
--- a/examples/mnist_mlp.py
+++ b/examples/mnist_mlp.py
@@ -12,10 +12,11 @@ from keras.datasets import mnist
 from keras.models import Sequential
 from keras.layers import Dense, Dropout
 from keras.optimizers import RMSprop
+import ngraph_bridge
 
 batch_size = 128
 num_classes = 10
-epochs = 20
+epochs = 1
 
 # the data, split between train and test sets
 (x_train, y_train), (x_test, y_test) = mnist.load_data()
