*** platform.h	2020-01-30 11:42:38.662817446 -0800
--- new_platform.h	2020-01-30 11:42:18.709151448 -0800
***************
*** 53,64 ****
  // compilation.
  //
  // TODO(b/138433137) Select AVX-512 at runtime rather than via compile options.
! #if defined(__AVX512F__) && defined(__AVX512DQ__) && defined(__AVX512CD__) && \
      defined(__AVX512BW__) && defined(__AVX512VL__)
! #define RUY_DONOTUSEDIRECTLY_AVX512 1
! #else
  #define RUY_DONOTUSEDIRECTLY_AVX512 0
! #endif
  
  // Detect APPLE
  #ifdef __APPLE__
--- 53,64 ----
  // compilation.
  //
  // TODO(b/138433137) Select AVX-512 at runtime rather than via compile options.
! //#if defined(__AVX512F__) && defined(__AVX512DQ__) && defined(__AVX512CD__) && \
      defined(__AVX512BW__) && defined(__AVX512VL__)
! //#define RUY_DONOTUSEDIRECTLY_AVX512 1
! //#else
  #define RUY_DONOTUSEDIRECTLY_AVX512 0
! //#endif
  
  // Detect APPLE
  #ifdef __APPLE__
